src/bootloader/startup.c

```diff
@@ -55,6 +56,7 @@ int main(void)
 #ifdef BOOTLOADER_DEVDEVICE
     qtouch_init();
 #endif
+    spi_mem_protected_area_lock();
```

Comment:
perhaps do it in `platform_init`? It should be conditional on `memory_get_platform() == MEMORY_PLATFORM_BITBOX02PLUS`